ntcore/src/main/native/cpp/LocalStorage.h

```diff
@@ -43,19 +36,46 @@ class LocalStorage final : public net::ILocalStorage {
       : m_impl{inst, listenerStorage, logger} {}
   LocalStorage(const LocalStorage&) = delete;
   LocalStorage& operator=(const LocalStorage&) = delete;
-  ~LocalStorage() final;
```

Comment:
Why was this removed in [[ntcore] Split LocalStorage implementation into separate files](https://github.com/wpilibsuite/allwpilib/pull/7216/commits/63e71a285125e35723998fe57883ee0f5d467eff)?

Reply:
It's no longer necessary.  The reason it was declared in the header was because we declared the Impl class in the .cpp file, which is no longer the case.